tests/integration/test_client_errors.py

```diff
@@ -111,21 +110,24 @@ def test_invalid_attribute_error(client, rand_gen):
     project.delete()
 
 
-@pytest.mark.slow
-# TODO improve consistency
-@pytest.mark.skip(reason="Inconsistent test")
-def test_api_limit_error(client, rand_gen):
-    project_id = client.create_project(name=rand_gen(str)).uid
+@pytest.mark.skip("timeouts cause failure before rate limit")
+def test_api_limit_error(client):
+    global limited
+    limited = False
 
     def get(arg):
         try:
-            return client.get_project(project_id)
+            return client.get_user()
         except labelbox.exceptions.ApiLimitError as e:
             return e
 
-    with Pool(300) as pool:
-        results = pool.map(get, list(range(2000)))
+    n = 1600
+    with Pool(30) as pool:
```

Comment:
Just wondering: where do 1600 / 30 come from? It might be worth to leave a comment

Reply:
Ok will do